### ISSUE TITLE
Malformed bower.json causes analysis to fail and retry

### ISSUE THREAD

**Issue by dstockwell** [webcomponents/beta#569]
```
$ bower install  link2twenty/l2t-context-menu#1.0.2
```

```
bower l2t-context-menu#1.0.2       EMALFORMED Failed to read /tmp/.../bower.json
```

**Comment by dstockwell** [comment]
Another case:

```
bower install rnicholus/file-input#1.0.0
```
```
bower file-input#1.0.0    invalid-meta The "main" field cannot contain globs (example: "*.js")
bower file-input#1.0.0           error Invalid Version: 1.0
```

**Comment by andymutton** [comment]
For the second one, that main thing is a warning, not an error. The error is "TypeError: Invalid Version: 1.0".

Easy to resolve both though.

**Comment by andymutton** [comment]
This should be fixed by #582 when it rolls out to beta.
